### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by antoniok-true** [NVIDIA/NVFlare#1240]
**Describe the bug**
After I install nvflare (via pip or docker) I get nvflare command not found.
running: `nvflare simulator -h`
gives: `nvflare: command not found`

**To Reproduce**
1. Go to ['nvflare getting started](https://nvflare.readthedocs.io/en/2.2.1/getting_started.html)
2. Follow instructions to install and run the package using pip install or the docker build method to install nvflare

**Desktop (please complete the following information):**
 - OS: MacOS
 - Python Version: 3.8.13
 - NVFlare Version: 2.1.2

**Comment by antoniok-true** [comment]
Thank you for your help with this Holger. I can't install those newer versions

```sh
ERROR: Ignored the following versions that require a different python version: 1.1.0 Requires-Python >=3.8,<3.9; 1.1.1 Requires-Python ==3.8.10; 2.1.0 Requires-Python >=3.7,<3.9; 2.1.1 Requires-Python >=3.7,<3.9; 2.1.2 Requires-Python >=3.7,<3.9; 2.1.3 Requires-Python >=3.7,<3.9; 2.1.4 Requires-Python >=3.7,<3.9; 2.2.0 Requires-Python >=3.7,<3.9; 2.2.0rc1 Requires-Python >=3.7,<3.9; 2.2.1 Requires-Python >=3.7,<3.9; 2.2.1rc1 Requires-Python >=3.7,<3.9; 2.2.1rc4 Requires-Python >=3.7,<3.9; 2.2.1rc5 Requires-Python >=3.7,<3.9; 2.2.1rc6 Requires-Python >=3.7,<3.9; 2.2.1rc7 Requires-Python >=3.7,<3.9; 2.2.1rc8 Requires-Python >=3.7,<3.9; 2.2.1rc9 Requires-Python >=3.7,<3.9; 2.2.2 Requires-Python >=3.7,<3.9; 2.2.3 Requires-Python >=3.7,<3.9; 2.2.4 Requires-Python >=3.7,<3.9
ERROR: Could not find a version that satisfies the requirement nvflare>=2.2.1 (from versions: 0.1.3, 0.9.0, 1.0.0, 1.0.1, 1.0.2, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.0.5, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.0.10, 2.0.11, 2.0.12, 2.0.13, 2.0.14, 2.0.15, 2.0.16, 2.0.18, 2.0.19)
ERROR: No matching distribution found for nvflare>=2.2.1
```

**Comment by holgerroth** [comment]
Did you try `pip install --upgrade pip`? `nvflare simulator` is not available in version <2.2, I believe. @IsaacYangSLA might be able to help here.

**Comment by antoniok-true** [comment]
Yes I tried `pip install --upgrade pip` I keep getting `ERROR: No matching distribution found for nvflare>=2.2.1`

**Comment by YuanTingHsieh** [comment]
Hi @antoniok-true 

Can you paste the following command's output here?

`python3 -V` or `python -V`


Once we make sure the python version is supported.
Then we can do

`python -m pip install --upgrade pip`

And `python -m pip install nvflare==2.2.4`

**Comment by antoniok-true** [comment]
Here it is:
```sh
(nvflare_cifar10) ➜  cifar10-sim git:(dev) ✗ python3 -V             
Python 3.9.16
(nvflare_cifar10) ➜  cifar10-sim git:(dev) ✗ python -V 
Python 3.9.16
```

And here's the pip upgrade response:
```sh
(nvflare_cifar10) ➜  cifar10-sim git:(dev) ✗ python -m pip install --upgrade 
pip
Requirement already satisfied: pip in ./nvflare_cifar10/lib/python3.9/site-packages (22.3.1)
```
And the pip install nvflare==2.2.4  
```sh                                                                     
(nvflare_cifar10) ➜  cifar10-sim git:(dev) ✗ 
python -m pip install nvflare==2.2.4                                         
(nvflare_cifar10) ➜  cifar10-sim git:(dev) ✗ python -m pip install nvflare==2
.2.4
ERROR: Ignored the following versions that require a different python version: 1.1.0 Requires-Python >=3.8,<3.9; 1.1.1 Requires-Python ==3.8.10; 2.1.0 Requires-Python >=3.7,<3.9; 2.1.1 Requires-Python >=3.7,<3.9; 2.1.2 Requires-Python >=3.7,<3.9; 2.1.3 Requires-Python >=3.7,<3.9; 2.1.4 Requires-Python >=3.7,<3.9; 2.2.0 Requires-Python >=3.7,<3.9; 2.2.0rc1 Requires-Python >=3.7,<3.9; 2.2.1 Requires-Python >=3.7,<3.9; 2.2.1rc1 Requires-Python >=3.7,<3.9; 2.2.1rc4 Requires-Python >=3.7,<3.9; 2.2.1rc5 Requires-Python >=3.7,<3.9; 2.2.1rc6 Requires-Python >=3.7,<3.9; 2.2.1rc7 Requires-Python >=3.7,<3.9; 2.2.1rc8 Requires-Python >=3.7,<3.9; 2.2.1rc9 Requires-Python >=3.7,<3.9; 2.2.2 Requires-Python >=3.7,<3.9; 2.2.3 Requires-Python >=3.7,<3.9; 2.2.4 Requires-Python >=3.7,<3.9
ERROR: Could not find a version that satisfies the requirement nvflare==2.2.4 (from versions: 0.1.3, 0.9.0, 1.0.0, 1.0.1, 1.0.2, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.0.5, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.0.10, 2.0.11, 2.0.12, 2.0.13, 2.0.14, 2.0.15, 2.0.16, 2.0.18, 2.0.19)
ERROR: No matching distribution found for nvflare==2.2.4
```

**Comment by YuanTingHsieh** [comment]
@antoniok-true thanks for pasting the python version.

The problem is that you have Python 3.9 and currently NVFlare requires the python version to be either 3.7 or 3.8.

You could try to install Python 3.8 in your system and use that at this very moment.

And we will make sure to support 3.8, 3.9 in next release. (3.10, 3.11 is pending discussion)

**Comment by YuanTingHsieh** [comment]
Actually @IsaacYangSLA can you update the pip upload process to specify we support 3.8, 3.9 and 3.10?

**Comment by pygabc1** [comment]
> ~Actually @IsaacYangSLA can you update the pip upload process to specify we support 3.8, 3.9 and 3.10?~
> 
> We've already fixed this in #1104 Next release this should be automatically solved.
> 
> For now you can either install python 3.8 then install NVFlare, or use python 3.9 with "dev" branch

Hi, my python version is 3.8.13, but I have same issue, which is
`python -m pip install nvflare==2
.2.3                                                                                                                                                                                                       
ERROR: Could not find a version that satisfies the requirement nvflare==2.3.0 (from versions: none)
ERROR: No matching distribution found for nvflare==2.3.0`
